include/mathfu/matrix.h

```diff
@@ -122,6 +122,13 @@ static inline Matrix<T, 4, 4> LookAtHelper(const Vector<T, 3>& at,
                                            const Vector<T, 3>& eye,
                                            const Vector<T, 3>& up,
                                            T handedness);
+template <class T>
+static inline bool UnProjectHelper(const Vector<T, 3>  windowCoord,
```

Comment:
We follow the google style guide https://google.github.io/styleguide/cppguide.html

windowCoord --> window_coord

same for other args.